discord/onboarding.py

```diff
@@ -0,0 +1,209 @@
+"""
+The MIT License (MIT)
+
+Copyright (c) 2015-present Rapptz
+
+Permission is hereby granted, free of charge, to any person obtaining a
+copy of this software and associated documentation files (the "Software"),
+to deal in the Software without restriction, including without limitation
+the rights to use, copy, modify, merge, publish, distribute, sublicense,
+and/or sell copies of the Software, and to permit persons to whom the
+Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+DEALINGS IN THE SOFTWARE.
+"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Optional, Set, List, Union
+
+from . import utils
+from .enums import OnboardingPromptType, try_enum
+from .utils import cached_slot_property
+
+__all__ = ('Onboarding', 'OnboardingPrompt', 'OnboardingPromptOption')
+
+
+if TYPE_CHECKING:
+    from .abc import GuildChannel
+    from .emoji import Emoji
+    from .guild import Guild
+    from .partial_emoji import PartialEmoji
+    from .role import Role
+    from .state import ConnectionState
+    from .threads import Thread
+    from .types.onboarding import (
+        Prompt as PromptPayload,
+        PromptOption as PromptOptionPayload,
+        Onboarding as OnboardingPayload,
+    )
+
+
+class OnboardingPromptOption:
+    """Represents an onboarding prompt option.
+
+    .. versionadded:: 2.2
+
+    Attributes
+    -----------
+    id: :class:`int`
+        The ID of this prompt option.
+    guild: :class:`Guild`
+        The guild the onboarding prompt option is related to.
+    title: :class:`str`
+        The title of this prompt option.
+    description: Optional[:class:`str`]
+        The description of this prompt option.
+    emoji: Optional[Union[:class:`Emoji`, :class:`PartialEmoji`, :class:`str`]]
+        The emoji tied to this option. May be a custom emoji, or a unicode emoji.
+    channel_ids: Set[:class:`int`]
+        The IDs of the channels that will be set visible if this option is selected.
+    role_ids: Set[:class:`int`]
+        The IDs of the roles given to the user if this option is selected
```

Comment:
```suggestion
    channel_ids: Set[:class:`int`]
        The IDs of the channels that will be made visible if this option is selected.
    role_ids: Set[:class:`int`]
        The IDs of the roles given to the user if this option is selected.
```
"set to visible" could also work but I feel like "made visible" sounds a bit better, perhaps?